src/api/network/asio/Imp.cpp

```diff
@@ -23,6 +23,7 @@
 #include <exception>
 #include <functional>
 #include <future>
+#include <iostream>
```

Comment:
is it needed?

Reply:
Probably no longer, now that the temporary diagnostics are gone.